app/server/versions/v1/db/init/optionsCollection.js

```diff
@@ -24,8 +24,8 @@ const options = [
     component: "post",
     options: {
       titleMinLength: 3,
-      titleMaxLength: 50,
-      titleDisallowedCharacters: "\\\\/@\\*\\^_\\+\\-=`~\\]\\[{}:;<>",
+      titleMaxLength: 200,
+      titleDisallowedCharacters: "\\\\/@\\*\\^_\\+\\-=`~\\]\\[{}:;<>#",
```

Comment:
I think this might not be working as expected? Can still enter some of these.
![image](https://user-images.githubusercontent.com/37922247/181077932-247a59ab-9106-4350-9dc6-7688fe834ae8.png)

Reply:
@dbarkowsky Are you testing on this branch? Everything pulled correctly?
![test](https://user-images.githubusercontent.com/16313579/181083630-3ce7f107-f563-48db-bce9-10d289dd89ed.PNG)

Reply:
Thought I had, probably my bad.